test/travis-before-install.sh

```diff
@@ -19,8 +19,15 @@ travis_retry go get \
   github.com/golang/lint/golint \
   github.com/mattn/goveralls \
   github.com/modocache/gover \
-  github.com/jcjones/github-pr-status \
-  github.com/letsencrypt/goose/cmd/goose
+  github.com/jcjones/github-pr-status &
+
+(wget https://github.com/jsha/boulder-tools/raw/master/goose.gz &&
```

Comment:
Good point. I still think it doesn't make sense to add a hash check for this one dependency unless we intend to add it for the others. What do you think? I'm assuming I don't currently have your r= because the PR doesn't meet the attached condition. :-)